### PR TITLE
Numerical statistics

### PR DESCRIPTION
**Pull request by GabrielKP**
Changes
* NumericalStatistics.hpp file with NumericalStatistics object, which stores a stats struct the NumericalStatistics object has a print function
* I have inserted this in Wrappers.hpp in the appropriate place
* NumericalStatistics.cpp file in the test folder, inserted in the automatic tests
* Tests all instances in test/instances for correctness of NumericalStatistics (written off by Gurobi)
* Uses test/instances/Instances.hpp to load all instances
* Each instance is under test/instances/instancename.hpp as hpp which you can include and then get the problem back with papilo::instances::instancename()
* I have tested each one, each one is equal to instances read from the MPSParser in the following aspects ( nRows, nCols, numIntegralCols, numContinuousCols, Objective:Coeff,offset, MatrixCoefficients, Rhs, Lhs, UBs, LBs, Colflags: kNone,kLbInf,kLbHuge,kUbInf,kUbHuge,kIntegral,kFixed,kSubstituted,kImplInt, Rowflags:None,kLhsInf,kRhsInf,kIntegral,kRedundant, ColNames and RowNames - exception: problemName is not equal! ), although I think that many of the flags will be set later it is a good sign that the MPSWriter and the ProblemBuilder problem are the same in these aspects
* The files are created with a script "src/convMPS.cpp" that prints the code into the console to create a problem with the ProblemBuilder
* convMPS.cpp is compiled as executable (probably suboptimal)
* ProblemBuilder.hpp has received a few updates: added functions that take a vector instead of a single column/row and update all rows
* In Problem.hpp I adjusted 5 comments (which seemed wrong to me)

### CODE REVIEW

**Comment by GabrielKP** [comment]
I have now added 2 checks for dynamism in the test for flugpl,I computed them by hand

File: src/papilo/core/ProblemBuilder.hpp [hunk 1]
```diff
@@ -161,6 +210,14 @@ class ProblemBuilder
          rflags[row].unset( RowFlag::kLhsInf );
    }
 
+   void
+   setRowLhsInfAll( Vec<bool> isInfinite )
```
Comment (lgottwald) [inline src/papilo/core/ProblemBuilder.hpp]: Do not use Vec\<bool\> (remember as a general rule of thumb in C++). The standard has made some weird exceptions for std::vector\<bool\> that make it not behave as a vector for other types. Rather use std::vector\<uint8_t\> or \<unsigned char\> or use a bitset e.g. from boost. In this case you could also use a template argument and not specify the exact type.

Comment (lgottwald) [reply src/papilo/core/ProblemBuilder.hpp]: this applies to all other places where you use vector\<bool\>

**Comment by lgottwald** [comment]
if you change this last thing with the Vec<bool> this can be merged. Sorry for the long time the review took.

**Review by lgottwald — APPROVED** [review]

**Pull request merged**